src/pdm/cli/commands/list.py

```diff
@@ -309,6 +309,19 @@ def parse_comma_separated_string(
     return items
 
 
+def _get_metadata_key(dist: im.Distribution, key: str) -> str | None:
+    """Get a key from metadata, or return None if it is not found or 'UNKNOWN'."""
+    # Starting with importlib-metadata 8.0, KeyError is thrown for missing keys.
+    try:
+        value = dist.metadata[key]
+    except KeyError:
+        return None
+    else:
+        if not value or value == "UNKNOWN":
```

Comment:
I would prefer to just leave `UNKNOWN` as-is? or why not?

Reply:
All the calling code in this file was replacing UNKNOWN with None, see further down in the diff.

Reply:
But agree that if we want a more general helper function, it's better to keep the "UNKNOWN" check out. I think I'll add a helper to `utils.py` and use that in the places this PR already changes plus a few more.